pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1015,6 +1015,16 @@ func validateNUMA(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec, con
 				Field: field.Child("domain", "cpu", "numa", "guestMappingPassthrough").String(),
 			})
 		}
+		if spec.Domain.CPU != nil && spec.Domain.CPU.Threads > 2 {
```

Comment:
Can we also add this validation to `validateCpuPinning`?

Reply:
Done.